src/features/transactions/swapEventSubscriber.ts

```diff
@@ -126,23 +20,62 @@ export default function subscribeToSwapEvents(params: {
   dispatch: Dispatch;
 }) {
   const { swapContract, account, dispatch, chainId } = params;
-  swapContract.on("Swap", async (...data) => {
-    const transactions = store.getState().transactions;
-    const { nonce, signerWallet, transactionHash, protocol, transaction } =
-      mapSwapEvent(data, chainId, account, transactions);
 
-    // If we don't have a 'transaction', we don't already know about this swap
-    // and therefore don't need to update the UI.
-    if (!transaction) return;
+  const _account = account.toLowerCase();
```

Comment:
Typescript type requires it to be a string - if someone passes a potentially null value elsewhere they'll be warned about it, so this is ok.

Reply:
Typescript type is `string` (not `string?` or `string | null`) so this can't be called with something that's potentially null without a typescript error.